package.json

```diff
@@ -1558,6 +1560,7 @@
     "val-loader": "^1.1.1",
     "vinyl-fs": "^4.0.0",
     "watchpack": "^1.6.0",
+    "web-streams-polyfill": "^3.2.1",
```

Comment:
The `devDependencies` `web-streams-polyfill` package is for the `ReadableStream` [polyfill](https://js.langchain.com/docs/get_started/installation#unsupported-nodejs-16), for unit tests